app/services/data_migrations/backfill_selected_boolean.rb

```diff
@@ -0,0 +1,12 @@
+module DataMigrations
+  class BackfillSelectedBoolean
+    TIMESTAMP = 20210517161321
+    MANUAL_RUN = false
+
+    def change
+      ApplicationReference.feedback_provided.find_each(batch_size: 100) do |reference|
+        reference.update!(selected: true)
```

Comment:
I think we can `find_in_batches` and then call `update_all` on each batch to keep memory usage and total number of SQL queries low